src/server/scripts/Commands/cs_debug.cpp

```diff
@@ -1900,7 +1683,7 @@ class debug_commandscript : public CommandScript
         return true;
     }
 
-    static bool HandleDebugOutOfBounds(ChatHandler* handler, CommandArgs* /*args*/)
+    static bool HandleDebugOutOfBounds(ChatHandler* handler)
```

Comment:
I fundamentally disagree with this function existing in our source code.

Reply:
I guess let's see what @jackpoz has to say

Reply:
too late e18f7aa